doc/neps/nep-0042-new-dtypes.rst

```diff
@@ -259,21 +259,41 @@ including the type hierarchy and the use of abstract DTypes.
 Class getter
 ==============================================================================
 
-To create a dtype instance from a scalar type users now call ``np.dtype`` (for
-instance, ``np.dtype(np.int64)``).
-
-To get the DType of a scalar type, we propose this getter syntax::
+To create a DType instance from a scalar type users now call
+``np.dtype`` (for instance, ``np.dtype(np.int64)``). Sometimes it is
+also necessary to access the underlying DType class; this comes up in
+particular with type hinting because the "type" of a DType instance is
+the DType class. Taking inspiration from type hinting, we propose the
+following getter syntax::
 
     np.dtype[np.int64]
 
-The notation works equally well with built-in and user-defined DTypes
-and is inspired by and potentially useful for type hinting.
+to get the DType class corresponding to a scalar type. (Note that with
+the acceptance of PEP 585 [2]_ there is precedent for
+``np.dtype[np.int64]`` referring to a *subclass* of ``np.dtype``.) The
+notation works equally well with built-in and user-defined DTypes.
 
 This getter eliminates the need to create an explicit name for every
-DType, crowding the ``np`` namespace; the getter itself signifies the type.
+DType, crowding the ``np`` namespace; the getter itself signifies the
+type. It also opens up the possibility of making ``np.ndarray``
+generic over DType class via annotations like::
```

Comment:
```suggestion
generic over DType class using annotations like::
```
Minor. [Google style guide](https://developers.google.com/style/word-list#letter-v) says of "via": "Don't use."